uber/site_sections/dealer_reports.py

```diff
@@ -28,7 +28,7 @@ def seller_table_info(self, out, session):
         ])
         dealer_groups = session.query(Group).filter(Group.tables > 0).all()
         for group in dealer_groups:
-            if group.approved and group.is_dealer:
+            if group.approved and group.status==c.APPROVED and group.is_dealer:
```

Comment:
I could totally see just ditching the group.approved condition in favor of the status check